app.js

```diff
@@ -0,0 +1,50 @@
+let products = [];
```

Comment:
this is a classic single-point-of-truth issue. you actually don't need this array if you use local storage